webapp/src/test/resources/io/github/microcks/util/openapi/pastry-with-proxy-openapi.yaml

```diff
@@ -0,0 +1,84 @@
+---
+openapi: 3.0.2
+info:
+  title: pastry-proxy
+  version: 1.0.0
+  description: API definition of API Pastry sample app
+  contact:
+    name: Laurent Broudoux
+    url: http://github.com/lbroudoux
+    email: laurent.broudoux@gmail.com
+  license:
+    name: MIT License
+    url: https://opensource.org/licenses/MIT
+paths:
+  /pastry/{name}:
+    summary: Global operations on pastries
+    get:
+      x-microcks-operation:
+        dispatcher: PROXY
+        dispatcherRules: http://localhost/rest/pastry-proxy/1.0.0/real-pastry/{{ request.path[2] }}
```

Comment:
That's not exactly what I had in mind... I was thinking of something even simpler where the `dispatcherRules` is just the base URL of a backend service that implements the same API. If reusing another mock (here called `real-backend`) served by Microcks, it would have done something like this: 

```yaml
 x-microcks-operation:
   dispatcher: PROXY
   dispatcherRules: http://localhost/rest/real-backend/1.0.0
```

Reply:
Thanks for this example! I've refactored both dispatcher according to this logic.